packages/synapse-react-client/src/lib/containers/download_list/DownloadConfirmation.tsx

```diff
@@ -296,45 +299,47 @@ export const DownloadConfirmation: React.FunctionComponent<
   return (
     <>
       <Alert
-        dismissible={false}
-        show={true}
-        variant={'info'}
-        transition={false}
+        severity={StatusConstruct[status].severity}
         className={`download-confirmation ${
-          StatusConstruct[status].className
```

Comment:
Change the alert styling with Alert's severity prop, rather than setting a className